projects/demo/src/modules/components/input-slider-legacy/index.html

```diff
@@ -0,0 +1,238 @@
+<tui-doc-page
+    header="InputSliderLegacy"
+    package="LEGACY"
+    type="components"
```

Comment:
please add "DEPRECATED" badge and link to the new variant

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/10288/commits/05c18430f6f4aa2c0d07784541f74f0b9202b87e